src/pages/journal/journal.effects.ts

```diff
@@ -32,8 +36,13 @@ export class JournalEffects {
       return this.journalProvider
         .getJournal(journal.lastRefreshed)
         .pipe(
-          map(journalData => newSlotsDetectingChanges(journal.slots, journalData)),
-          map(slots => new journalActions.LoadJournalSuccess(slots)),
+          map((journalData: ExaminerWorkSchedule) => this.slotProvider.detectSlotChanges(journal.slots, journalData)),
+          map((slots: any[]) => {
+            const s = groupBy(slots, this.slotProvider.getSlotDate);
+            console.log('slots', s);
```

Comment:
Log line to be removed + map block can be onelined into return value.

Reply:
ahh thanks for pointing it out. Thought I got rid of all of them :D